WordPressAuthenticatorTests/Mocks/MockWordpressAuthenticatorProvider.swift

```diff
@@ -0,0 +1,101 @@
+@testable import WordPressAuthenticator
+
+struct MockWordpressAuthenticatorProvider {
```

Comment:
I find the name "mock" here misleading. In most literature, like [here](http://xunitpatterns.com/Mock%20Object.html), mock is a test double.

When I read in the test that "mock provider, get authenticator" was used, the first thing I thought was that the test was checking the behavior of the mock test double instead of the production class.

This type returns actual production code instances, not test doubles.

How do you feel about the name `WordpressAuthenticatorProvider`? Or `WordpressAuthenticatorProviderForTests` to make it clearer that these are preconfigured instances for the tests?